### ISSUE TITLE
Recruitment Optimization Stack

### ISSUE THREAD

**Issue by ThaliaBarrera** [airbytehq/quickstarts#23]
### Optimizing Recruitment Processes with Applicant Data.

Extract data from Applicant Tracking Systems using Airbyte, transform it with dbt, and analyze to optimize recruitment processes.

**Comment by AnkitAnandMastery17** [comment]
Hi @ThaliaBarrera , I want to work on this issue. Please give me more instructions as I am new to open source.

**Comment by adnankaya** [comment]
Hi! Can I work on this?

**Comment by rohinish404** [comment]
Hey, i would like to work on this! Any resources where can i get started ?

**Comment by ThaliaBarrera** [comment]
Hi @rohinish404! Sure, I have assigned you to it.

**Comment by ThaliaBarrera** [comment]
Hi @rohinish404, this message is just to remind you that according to the Hackathon rules you have 5 days to submit progress on the issue. I’ll give you a couple more days to submit a PR, otherwise I’ll have to mark the issue as “To be Claimed” or reassign it. 

Thank you for understanding! Let me know if you have any questions.

**Comment by rohinish404** [comment]
@ThaliaBarrera Yeah you can unassign me.. i don't think i will be able to give time to this. :)

**Comment by Terieyenike** [comment]
Can I give this a shot? Please share resources for reference.

Thanks

**Comment by rajeshkhadka200** [comment]
Hey @ThaliaBarrera , I want to work on this issue, could you please assign it to me ??

**Comment by sharsha315** [comment]
Hi @ThaliaBarrera , can I have the opportunity to work on this issue?

**Comment by ThaliaBarrera** [comment]
Hi @Terieyenike, I have assigned the issue to you. For reference, you can check the README in this repo, as well as the existing quickstarts. I recommend implementing one of them, for example, the e-commerce analytics one.

**Comment by Terieyenike** [comment]
Thank you, @ThaliaBarrera

**Comment by ThaliaBarrera** [comment]
I am unassigning this issue as there has been no progress in the last 5 days. This is now open!

**Comment by sharsha315** [comment]
Hi @ThaliaBarrera, can I have the opportunity to work on this issue?

**Comment by ThaliaBarrera** [comment]
Hi @sharsha315! Sure, I have assigned it to you. Let me know if you have any questions.

**Comment by sharsha315** [comment]
Hi @ThaliaBarrera, Thank you for assigning the issue to me. From one of the comments above, I see that you have mentioned about the resources and references, I believe that is the place to start.

**Comment by ThaliaBarrera** [comment]
Yes, @sharsha315 that's the best place to start.

**Comment by sharsha315** [comment]
Hi @ThaliaBarrera, a little help here. I'm wondering if Google Cloud BigQuery should be used for this issue, like in the quickstart example above.

**Comment by ThaliaBarrera** [comment]
You can use any data warehouse you'd like @sharsha315 :) it can be BigQuery.

**Comment by ThaliaBarrera** [comment]
Hi @sharsha315! Are you still working on this? Otherwise I may need to unassign. Let me know :)

**Comment by sharsha315** [comment]
Hi @ThaliaBarrera, I apologize for not completing the task as initially intended due to unforeseen commitments. I kindly request to be unassigned from this issue to allow another contributor to take over while I continue to support the project in other capacities. Thank You.

**Comment by ThaliaBarrera** [comment]
Thanks for letting me know @sharsha315!
